### DISCUSSION TITLE
API Playground Language autodetection? How to?

### DISCUSSION THREAD

**Discussion by altryne** [deepgram/community#148] [General help] [answered]
Hi, as announced here, Deepgram now has language autodetection? 

https://blog.deepgram.com/nextgen-speaker-diarization-and-language-detection-models

I would like to play around with testing this, as I often translate audio with no language specified, how to I turn this on? Is this available in playground already?

**Comment by SandraRodgers** [comment, marked as answer]
Hi @altryne ,

If you go to the [Playground](https://playground.deepgram.com/) and look at the very top, under where it says Deepgram API Playground, you will see a language dropdown menu. Click on that and the very first selection is Automatic Language Detection.

<img width="811" alt="Screen Shot 2023-05-12 at 8 42 39 AM" src="https://github.com/deepgram/community/assets/45321563/b525aa2b-03e8-404d-a48d-b9c2557967fb">


Try it out and let us know what you think! Thanks!

Sandra

**Reply by altryne** [reply]
Thanks @SandraRodgers 

This notification was confusing, figured I'd ask! 

<img width="679" alt="image" src="https://github.com/deepgram/community/assets/463317/b848c413-3235-450b-aba3-6d20a62c0cf6">

**Comment by jpvajda** [comment]
@altryne  👋  Thanks for pointing this out to us, we appreciate it!  We recently introduced more language options into the API Playground so the beta banner is a bit outdated. We are going to update that text to be more clear.

Just let us know if you have any other feedback on it.
